docs/getting-started.md

```diff
@@ -10,27 +10,31 @@ aliases: ["/docs/"]
 
 ## Pre-requisites
 
-Before you get started:
+Before you get started using KUDO, you need to have a running Kubernetes cluster setup. You can use Minikube for testing purposes.
 
-- Kubernetes Cluster `1.13` or later (e.g. [Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/))
-- [Install kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) with version `1.13` or later
+- Setup a Kubernetes Cluster in version `1.13` or later (e.g. [Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/))
+- Install [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) in version `1.13` or later.
 
-## Install KUDO into your cluster
+### Notes on Minikube
 
-- `kubectl create -f https://raw.githubusercontent.com/kudobuilder/kudo/v0.5.0/docs/deployment/00-prereqs.yaml`
-- `kubectl create -f https://raw.githubusercontent.com/kudobuilder/kudo/v0.5.0/docs/deployment/10-crds.yaml`
-- `kubectl create -f https://raw.githubusercontent.com/kudobuilder/kudo/v0.5.0/docs/deployment/20-deployment.yaml`
+If you plan on developing and testing KUDO locally via Minikube, you'll need to launch your cluster with a reasonable amount of memory allocated. By default, Minikube runs with 2GB - we recommend at least 8GB, especially if you're working with applications such as [Kafka](/docs/examples/apache-kafka/). You can start Minikube with some suitable resource adjustments as follows:
 
-If you want to use the KUDO kubectl plugin, you can now follow the [CLI plugin installation instructions](https://kudo.dev/docs/cli/).
+```bash
+minikube start --cpus=4 --memory=10240 --disk-size=40g
+```
 
-### Notes on Minikube
+## Install KUDO into your cluster
```

Comment:
I actually preferred the order as it was before, I think we really want to highlight how to install kudo into the cluster on that page, minikube to be is kind of secondary.